Riot/Modules/Common/Recents/RecentsViewController.m

```diff
@@ -120,7 +120,12 @@ - (void)finalizeInit
     tableSearchBar = [[UISearchBar alloc] initWithFrame:CGRectMake(0, 0, 600, 44)];
     tableSearchBar.autoresizingMask = UIViewAutoresizingFlexibleWidth;
     tableSearchBar.showsCancelButton = NO;
-    tableSearchBar.placeholder = [VectorL10n searchDefaultPlaceholder];
+    tableSearchBar.placeholder = [VectorL10n searchFilterPlaceholder];
+    UIImage *filterIcon = [UIImage imageNamed:@"filter_off"];
```

Comment:
We should really expose generated images to objc like we do for the strings.

Reply:
I made a dedicated PR for this : https://github.com/vector-im/element-ios/pull/5533